### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by shivapbhusal**
Please find the plugin for Lock Screen detection.

### CODE REVIEW

File: .idea/copyright/epl_v10.xml [hunk 1]
```diff
@@ -1,6 +0,0 @@
-<component name="CopyrightManager">
```
Comment (fgwei) [inline .idea/copyright/epl_v10.xml]: Don't delete this file. You can use `git checkout .idea` to recover this file.

File: .idea/copyright/profiles_settings.xml [hunk 2]
```diff
@@ -1,7 +0,0 @@
-<component name="CopyrightManager">
```
Comment (fgwei) [inline .idea/copyright/profiles_settings.xml]: Same as above

File: .idea/uiDesigner.xml [hunk 3]
```diff
@@ -1,124 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (fgwei) [inline .idea/uiDesigner.xml]: Same as above

File: org.argus.amandroid.concurrent/src/main/main3.iml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (fgwei) [inline org.argus.amandroid.concurrent/src/main/main3.iml]: Why this file is edited?

File: org.argus.amandroid.concurrent/src/test/test1.iml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (fgwei) [inline org.argus.amandroid.concurrent/src/test/test1.iml]: Why this file is edited?

File: org.argus.amandroid/src/main/main1.iml [hunk 6]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (fgwei) [inline org.argus.amandroid/src/main/main1.iml]: Why this file is edited?

File: org.argus.amandroid/src/test/test2.iml [hunk 7]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (fgwei) [inline org.argus.amandroid/src/test/test2.iml]: Why this file is edited?

File: org.argus.jawa/org.argus.jawa.iml [hunk 8]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (fgwei) [inline org.argus.jawa/org.argus.jawa.iml]: Why this file is edited?

File: org.argus.jawa/src/main/main.iml [hunk 9]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (fgwei) [inline org.argus.jawa/src/main/main.iml]: Why this file is edited?

File: org.argus.jawa/src/test/resources/resources.iml [hunk 10]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (fgwei) [inline org.argus.jawa/src/test/resources/resources.iml]: Why this file is edited?

File: org.argus.jawa/src/test/test.iml [hunk 11]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (fgwei) [inline org.argus.jawa/src/test/test.iml]: Why this file is edited?

File: src/main/main2.iml [hunk 12]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (fgwei) [inline src/main/main2.iml]: Why this file is edited?

**Comment by shivapbhusal** [comment]
@fgwei , Thanks for the comment. Those files may have been deleted / modified automatically. I will make the necessary changes and send the pull request again.

**Pull request closed without merging**